.github/workflows/build.yml

```diff
@@ -1,7 +1,7 @@
 name: Build ASUS images
 on:
   schedule:
-    - cron: '00 08 * * *'  # 8:00am everyday
+    - cron: '45 15 * * *'  # 15:45 UTC everyday
```

Comment:
I thought we agreed to 1540, not 45?

https://github.com/ublue-os/main/issues/363#issuecomment-1788338503